### PR TITLE
freebooter kitchen additions & broken turf icons fixes

### PR DESCRIPTION
**Pull request by RustingWithYou**
The Freebooter ship kitchen now has salt, pepper and space spice so they can do more cooking. Also fixes some broken turf icon states because they bugged me.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

**Comment by RustingWithYou** [comment]
!review

File: maps/away/ships/freebooter/freebooter_ship.dmm [hunk 1]
```diff
@@ -955,9 +926,7 @@
 	layer = 2.31;
 	name = "lattice"
 	},
-/turf/simulated/floor{
-	icon_state = "dmg2"
```
Comment (DreamySkrell) [inline maps/away/ships/freebooter/freebooter_ship.dmm:959]: image

Floor damage was changed from being icon_states to decals - use the decals above to add back the damaged feel to the floors that had it before.

**Review by DreamySkrell — CHANGES_REQUESTED** [review]

**Comment by NonQueueingMatt** [comment]
Reopen when changes required are resolved

**Pull request closed without merging**